This is currently throwing errors:

```
ERROR: test_server (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_server
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/Users/sna219/Documents/School/DevOps/nyu-recommendations-s17/test_server.py", line 6, in <module>
    import server
  File "/Users/sna219/Documents/School/DevOps/nyu-recommendations-s17/server.py", line 17, in <module>
    from flask import Flask, Response, jsonify, request, json
ImportError: No module named flask
```

--- a/Vagrantfile
+++ b/Vagrantfile
@@ -74,6 +74,7 @@ Vagrant.configure(2) do |config|
   config.vm.provision "shell", inline: <<-SHELL
     # Import test data
     cd /vagrant
+    sleep 30
     sudo docker cp recommendations.sql mysql:/recommendations.sql
     sudo docker exec -i mysql mysql -h 127.0.0.1 -u root < recommendations.sql
   SHELL

--- a/test_server.py
+++ b/test_server.py
@@ -18,6 +18,13 @@ HTTP_409_CONFLICT = status.HTTP_409_CONFLICT
 #  T E S T   C A S E S
 ######################################################################
 class TestRecommendationServer(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        server.inititalize_mysql()
+
+    @classmethod
+    def tearDownClass(self):
+        server.conn.close()
 
     def setUp(self):
         server.app.debug = True

